Update (#433)
flarum/docs: docs/contributing.md

git clone https://github.com/flarum/flarum.git
cd flarum

> # Or, when you want to clone in the actually directory

Reviewer: ```suggestion
# Or, when you want to clone directly into the current directory
```

```bash
cd packages/framework

> yarn

Reviewer: I think it's better to recommend using the full command, even though yarn on its own does the same thing.

```suggestion
yarn install
```

You represent that the Contributions are Your original works of authorship, and to Your knowledge, no other person claims, or has the right to claim, any right in any invention or patent related to the Contributions. You also represent that You are not legally obligated, whether by entering into an agreement or otherwise, in any way that conflicts with the terms of this license.

> The Flarum Foundation acknowledges that, except as explicitly described in this Agreement, any Contribution which you provide is on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING, WITHOUT LIMITATION, ANY WARRANTIES OR CONDITIONS OF TITLE, NON-INFRINGEMENT, MERCHANTABILITY, OR FITNESS FOR A PARTICULAR PURPOSE.

Reviewer 2: What is the change here? This seems to be part of the MIT license and it shouldn't be modified.

Reviewer: New line at the end of the file.

This matches with what we have in every other repository.
